### DISCUSSION TITLE
Android and IPSec pre-shared key (IPSec XAuth PSK)

### DISCUSSION THREAD

**Discussion by adepaoli** [strongswan/strongswan#698] [Q&A]
Could it be possible to add the support to IPSec Xauth PSK?
I know that it's the least secure mode of running IKE/IPsec but there are many old Fortigate (or other brands) Firewalls around that don't support IKEv2/IPsec but only XAuth with a pre-shared key.
Any suggestion?

**Comment by tobiasbrunner** [comment]
No, the Android app does not and never will support IKEv1.

**Reply by adepaoli** [reply]
Thanks, If I would like to add MDM and IPSEC/IKEv1 support how we could collaborate together?

**Reply by tobiasbrunner** [reply]
What do you mean? And regarding IKEv1, that will never get added to the app and I will not provide any help on that.
